state-chain/node/src/chain_spec/berghain.rs

```diff
@@ -52,6 +53,37 @@ pub const ENV: StateChainEnvironment = StateChainEnvironment {
 	sol_vault_address: SolAddress([0; 32]), // TODO: fill in the valid Solana address,
 	sol_genesis_hash: Some(SolHash(bs58_array("5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d"))),
 	sol_usdc_address: SolAddress(bs58_array("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v")),
+	// todo: put correct values for nonces and accounts
```

Comment:
Better to make this a todo!() 

That way if we ever do create a new mainnet chainspec (unlikely), we *can't* forget to set these values.

Reply:
we can't have a todo!() here since this is a const and so it is evaluated at compile time and so it errors during compilation